runtimes/eden/src/migrations.rs

```diff
@@ -0,0 +1,57 @@
+use crate::Runtime;
+use frame_support::{migration, traits::OnRuntimeUpgrade, weights::Weight};
+
+#[cfg(feature = "try-runtime")]
+use codec::{Decode, Encode};
+#[cfg(feature = "try-runtime")]
+use frame_support::Blake2_128Concat;
+#[cfg(feature = "try-runtime")]
+use pallet_uniques::CollectionDetails;
+#[cfg(feature = "try-runtime")]
+use primitives::{AccountId, Balance};
+#[cfg(feature = "try-runtime")]
+use sp_std::prelude::*;
```

Comment:
If you want to drop the multiple lines of feature flags, I believe you can do something like the below
```suggestion
#[cfg(feature = "try-runtime")]
use {
  codec::{Decode, Encode},
  frame_support::Blake2_128Concat,
  pallet_uniques::CollectionDetails,
  primitives::{AccountId, Balance},
  sp_std::prelude::*
};
```